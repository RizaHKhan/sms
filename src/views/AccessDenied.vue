<template>
  <div>
    <h1>AccessDenied</h1>
  </div>
</template>

<script>
export default {
  name: 'AccessDenied'
};
</script>
