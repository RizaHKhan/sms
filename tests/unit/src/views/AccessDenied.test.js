import AccessDenied from '@/views/AccessDenied.vue';

import testHelpers from '@@/testHelpers.js';

describe('AccessDenied.vue', () => {
  let localVue;
  let store;
  let setup;

  let permissions = ['unassigned'];

  beforeEach(() => {
    setup = testHelpers.setupTestVariables(permissions);
    localVue = setup.localVue;
    store = setup.store;
  });

  test('Renders correctly', async () => {
    const wrapper = await testHelpers.shallowMountWrapper(AccessDenied, {
      localVue,
      store
    });

    expect(wrapper)
      .toMatchSnapshot();
  });
});
