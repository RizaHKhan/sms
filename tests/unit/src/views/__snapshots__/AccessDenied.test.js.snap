// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccessDenied.vue Renders correctly 1`] = `
<div>
  <h1>AccessDenied</h1>
</div>
`;
